app/controllers/patients_controller.rb

```diff
@@ -20,6 +20,14 @@ def show
 
     @jurisdiction_path = @patient.jurisdiction_path
 
+    @possible_jurisdiction_paths = if current_user.can_transfer_patients?
```

Comment:
is this class variable being used? I can't seem to find any references to it

Reply:
Great catch, fixed.